src/core/server/elasticsearch/elasticsearch_client_config.ts

```diff
@@ -28,7 +28,7 @@ import { Logger } from '../logging';
 import { ElasticsearchConfig } from './elasticsearch_config';
 
 /**
- * @internalremarks Config that consumers can pass to the Elasticsearch JS client is complex and includes
+ * @privateRemarks Config that consumers can pass to the Elasticsearch JS client is complex and includes
```

Comment:
didn't find `@internalremarks` in spec https://github.com/microsoft/tsdoc/blob/master/tsdoc/src/details/StandardTags.ts